### PR TITLE
chore: prepare for release

### PR DESCRIPTION
**Pull request by npmccallum**
Two commits in preparation for release.

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1798,33 +1807,22 @@ dependencies = [
 name = "sev_attestation"
 version = "0.1.0"
 dependencies = [
- "sallyport",
+ "sallyport 0.3.0",
```
Comment (bstrie) [inline Cargo.lock:1810]: Looks like sev_attestation still needs updating.

**Comment by bstrie** [comment]
I thought we disabled the action that checked the PR title?

File: Cargo.lock (hunk 1 above)
Comment (npmccallum) [reply Cargo.lock]: I don't see this in my latest push.

**Review by bstrie — APPROVED** [review]

**Comment by bstrie** [comment]
The SGX builder has been randomly failing with that error all day, re-running the failed actions tends to fix it. I told Paul a while back, not sure if he's looked into it yet.

**Pull request merged**